src/layer/dataset.ts

```diff
@@ -148,8 +148,8 @@ export const DATASET_EOCLOUD_LANDSAT7: Dataset = {
   searchIndexUrl: 'https://eocloud.sentinel-hub.com/index/landsat7/v2/search',
   findDatesUTCUrl: 'https://eocloud.sentinel-hub.com/index/landsat7/v2/dates',
   orbitTimeMinutes: 99,
-  minDate: new Date(Date.UTC(1999, 1 - 1, 1, 0, 0, 0)),
-  maxDate: new Date(Date.UTC(2003, 12 - 1, 1, 23, 59, 59)),
+  minDate: new Date(Date.UTC(1999, 7 - 1, 3, 19, 16, 17)), // 1999-07-03T19:16:17.163
+  maxDate: new Date(Date.UTC(2017, 1 - 1, 15, 23, 49, 14)), // 2017-01-15T23:49:14.495
```

Comment:
Should here be 1 second later (15 instead of 14)? Similar to `DATASET_EOCLOUD_ENVISAT_MERIS` and `DATASET_EOCLOUD_LANDSAT5`, where the second for `maxDate` is rounded upwards.

Reply:
Thanks, fixed!

With Meris and Landsat 5 it seems correct to me, but maybe it was fixed with other commits already.

Reply:
It seems correct to me as well. Will resolve.